airbyte-integrations/connectors/destination-postgres/src/integrationTestLegacy/kotlin/io/airbyte/integrations/destination/postgres/typing_deduping/PostgresRawOverrideDisableTypingDedupingTest.kt

```diff
@@ -17,11 +19,31 @@ class PostgresRawOverrideDisableTypingDedupingTest : PostgresTypingDedupingTest(
     override val rawSchema: String
         get() = "overridden_raw_dataset"
 
+    // when disable_type_dedupe = true, we only output to raw table
     override fun disableFinalTableComparison(): Boolean {
         return true
     }
 
+    override fun disableRawTableComparison(): Boolean {
+        return false
+    }
+
     @Disabled @Test override fun identicalNameSimultaneousSync() {}
 
     @Disabled @Test override fun testVarcharLimitOver64K() {}
+
+    // this test assumes that fields that are not in the schema will show up
+    // on the raw table. This is only true for older versions of the connector.
+    @Disabled @ParameterizedTest @ValueSource(longs = []) override fun testIncrementalSyncDropOneColumn(inputGenerationId: Long) {}
+
+    // this test assumes that the fully qualified raw table name is lowercased.
+    // This was only a restriction in older versions of the connector.
```

Comment:
I don't really know why we had this restriction though

Reply:
this makes me nervous - is this saying that where previously we did `create table "airbyte_internal"."blah_raw__stream_the_table"`, we're now doing `airbyte_internal."blah_raw__stream_The_Table"`? (assuming the stream name is `The_Table`)

wouldn't this require users to rewrite their SQL queries to target the mixed-case table?

([these stats](https://docs.google.com/document/d/1c2BFMiC9gHPg9gxPOhLABErsGZL8R6UZx89oOzeMDEM/edit?tab=t.0#heading=h.4jshuc33jnxr) are about a year old, but it looks like around 1/3 of cloud destination-postgres users have T+D disabled today, so that's pretty significant impact)